app/src/main/java/co/tinode/tindroid/CallFragment.java

```diff
@@ -280,6 +303,22 @@ private static VideoCapturer createCameraCapturer(CameraEnumerator enumerator) {
         return null;
     }
 
+    private void muteVideo() {
+        try {
+            mVideoCapturerAndroid.stopCapture();
+            mLocalVideoView.setVisibility(View.INVISIBLE);
+            sendToPeer(VIDEO_MUTED_EVENT);
+        } catch (InterruptedException e) {
+            Log.d(TAG, e.toString());
+        }
+    }
+
+    private void unmuteVideo() {
+        mVideoCapturerAndroid.startCapture(1024, 720, 30);
```

Comment:
Use constants instead of literals? Please add a comment explaining why these values.

Reply:
Done.